### DISCUSSION TITLE
Martini parameters for RNA-lipid conjugate

### DISCUSSION THREAD

**Discussion by mandar5335** [Martini-Force-Field-Initiative/MartiniDiscussions#8] [Other molecules]
Hi,

Is it possible to build and simulate RNA-lipid conjugate (user defined lipid and standard RNA nucleotides) with Martini? 

What is general approach to build parameters at the point of conjugation? 

Thanks,
Mandar

**Comment by csbrasnett** [comment]
Hi Mandar,

I don't think there are currently any available models out of the box for such a construct. To parameterise it yourself, the recent paper on [protein lipidations](https://pubs.acs.org/doi/10.1021/acs.jctc.3c00604) might be a useful reference to follow. The approach would generally follow from the [tutorials](https://cgmartini.nl/docs/tutorials/) we have available about parameterising models for Martini.

**Reply by mandar5335** [reply]
Thanks for your reply @csbrasnett ; the paper gives me some idea about parameterization.
